src/__tests__/SignerAlgorithm-test.ts

```diff
@@ -1,14 +1,14 @@
 import SignerAlgorithm from '../SignerAlgorithm'
 import { toSignatureObject } from '../VerifierAlgorithm'
-import SimpleSigner from '../SimpleSigner'
-import EllipticSigner from '../EllipticSigner'
-import NaclSigner from '../NaclSigner'
+import SimpleSigner from '../signers/SimpleSigner'
```

Comment:
I would not expect any of these tests to change if these Signers were simple deprecated. What functional changes cause the tests changes here?

Reply:
The private key "fix" forced all the signatures to change.
I can revert the key and use a leading zero.

Reply:
ok, thanks

Reply:
fixed.
I reverted the key and added the leading zero.
Signatures went back to the original form.

Reply:
Sounds good. I think it's totally fine to push that update in a separate PR, but in this context I was confused.